source/assets/sass/vendor/_syntax.scss.erb

```diff
@@ -1 +1,3 @@
+# @@@ I don't think this file is used?
```

Comment:
We should confirm, and either remove it entirely - or figure out what it was doing, if anything.

Reply:
I'll confirm, either online or at the meeting, and proceed accordingly.

Reply:
I'm pretty confident it's not used. It looks like this was the default Rouge styles, but then it was replaced with the updated ones in `_syntax-accessible.scss`. This file is not imported anywhere.

See: https://github.com/sass/sass-site/commit/b1e62bc77b72c6229a5cc4ea6a365e391e45a00d